components/tracing/otel-collector/base/otel-collector.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: argoproj.io/v1alpha1
+kind: ApplicationSet
```

Comment:
I am a bit confused about the fact that we have an applicationSet here in the component directory.

The pattern is within argo-cd-apps folder, we have an applicationSet that will create Applications and each Application will target a specific cluster and the content of the Application is a folder in components where we have the k8s resources to deploy on that cluster part of that application.

Reply:
Yeah, I'm struggling a bit with the Helm chart and getting proper values passed into it.  I think I know the direction to go; the various layers make this a bit challenging.